packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -240,6 +294,35 @@ class ToOOXMLVisitor {
 
                             this.globalOOXML += ooxml;
                             this.tags = [];
+                        } else if (this.getClass(subNode) === TRANSFORMED_NODES.optional) {
+                            if (!subNode.hasSome) {
+                                this.isTraversingFalseCondition = true;
+                            }
+                            // traverse the whenSome properties now
+                            this.traverseNodes(subNode.whenSome, properties, true);
+
+                            if (!subNode.hasSome) {
+                                this.isTraversingFalseCondition = false;
+                            } else {
+                                this.isTraversingFalseCondition = true;
+                            }
+                            this.isTraversingElseCondition = true;
+                            // traverse whenNone properties now
+                            this.traverseNodes(subNode.whenNone, properties, true);
+                            let ooxml = '';
+                            for (let tag of this.conditionalOrOptionalTags) {
```

Comment:
It's better to use `const` inside for-loops because you don't want side-effects.